### DISCUSSION TITLE
Cache for harmonized PGS/score files as well as genotypes? (2.0.0-alpha5)

### DISCUSSION THREAD

**Discussion by bgulko** [PGScatalog/pgsc_calc#279] [Ideas]
I am running whole genome with a lot of 1KGP gVCF data, and applying a variety of score files harmonized to grch38 (AWS/singularity). I find that a large fraction of the wall-runtime is spent downloading these files, but I already have them (periodically updated). Perhaps I've missed it (if so my sincere apologies) but is there a way to offer pgsc_calc  a cache directory for harmonized scores separately from the cache file for genotypes / Ancestry?

I looked, but my apologies if this exists and I missed it.
--Brad

**Comment by smlmbrt** [comment]
Hi @bgulko, no there's no way to do that within the pipeline [yet], but the pgscatalog_utils are setup to only re-download scores that haven't changed. A workaround is to use the package to download the scoring files to a local directory, and then use that as input to the calculator.
